### PR TITLE
[chore] Use prettier for integration tests

### PR DESCRIPTION
**Pull request by sdbondi**
Standard formatting (almost exactly like `rust-fmt`) for js

### CODE REVIEW

**Review by delta1 — COMMENTED** [review]
Nice desperately needed. We also need the check done on commit and on CI but that can be additional. Maybe an update to the readme to indicate that `npm run fmt` should be run before commit and/or prettier integration used in IDE on save?

File: integration_tests/.prettierrc.yaml [hunk 1]
```diff
@@ -0,0 +1,6 @@
+---
+trailingComma: "all"
+tabWidth: 4
+semi: true
+singleQuote: true
+printWidth: 150
```
Comment (delta1) [inline integration_tests/.prettierrc.yaml]: I would much rather use the default options === nothing to bikeshed

File: integration_tests/features/support/world.js [hunk 2]
```diff
@@ -1,8 +1,8 @@
 const { setWorldConstructor, After, BeforeAll } = require("cucumber");
 
-const BaseNodeProcess = require('../../helpers/baseNodeProcess');
-const MergeMiningProxyProcess = require('../../helpers/mergeMiningProxyProcess');
-const WalletProcess = require('../../helpers/walletProcess');
+const BaseNodeProcess = require("../../helpers/baseNodeProcess");
+const MergeMiningProxyProcess = require("../../helpers/mergeMiningProxyProcess");
+const WalletProcess = require("../../helpers/walletProcess");
```
Comment (delta1) [inline integration_tests/features/support/world.js]: sigh javascript, if `singleQuote` is true then why are these double quotes?

File: integration_tests/package.json [hunk 3]
```diff
@@ -16,7 +18,8 @@
     "cucumber-pretty": "^6.0.0",
     "grpc": "^1.24.3",
     "grpc-promise": "^1.4.0",
-    "mocha": "^8.1.3"
+    "mocha": "^8.1.3",
```
Comment (delta1) [inline integration_tests/package.json]: `mocha` can be removed now afaik it's unused

**Comment by sdbondi** [comment]
Good ideas - done

File: integration_tests/.prettierrc.yaml (hunk 1 above)
Comment (sdbondi) [reply integration_tests/.prettierrc.yaml]: Haha yeah I agree

**Comment by delta1** [comment]
> Good ideas - done

sorry about the timing of these comments

File: integration_tests/features/support/world.js (hunk 2 above)
Comment (sdbondi) [reply integration_tests/features/support/world.js]: I want to blame js (really) but I think this one was me

Comment (delta1) [reply integration_tests/features/support/world.js]: I don't know, I ran the check and fmt locally and it stayed the same lol

Comment (sdbondi) [reply integration_tests/features/support/world.js]: Shakes head at JS... typical

**Pull request merged**